lnbits/app.py

```diff
@@ -146,9 +146,9 @@ async def vendored_assets_variable():
 
 
 def register_async_tasks(app):
-    @app.route("/wallet/webhook")
-    async def webhook_listener():
-        return await webhook_handler()
+    @app.api_route("/wallet/webhook", methods=["GET", "POST"])
+    async def webhook_listener(request: Request):
+        return await webhook_handler(request)
```

Comment:
I'm not python fluent, so maybe I'm missing something obvious, but webhooks for Lnpay didn't work for me (cannot remember the exact error message I got). 
I thought it was because of this: 

![image](https://user-images.githubusercontent.com/3494837/182039354-08e16cf7-2187-42e1-8712-db67d751caba.png)

Maybe it was only a local issue on my machine because of some different package versions or something else.

Reply:
I think that even if there is an issue here, it should be a separate PR. It doesn't seem to affect the strike extension.

Reply:
I have a commit to undo this. @Marfusios could you give me push access on your forked repo?

Reply:
Sure, an invitation sent

Reply:
thank you. pushed.